/home/nianliu/ModelGarden/models-r2.1.0/official/modeling/hyperparams/params_dict.py:402: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  params_dict = yaml.load(dict_or_string_or_yaml_file)
I0817 01:48:55.161027 140143557838656 main.py:195] Model Parameters: {'anchor': {'anchor_size': 4.0,
            'aspect_ratios': [1.0, 2.0, 0.5],
            'max_level': 7,
            'min_level': 3,
            'num_scales': 3},
 'architecture': {'backbone': 'resnet',
                  'multilevel_features': 'fpn',
                  'parser': 'retinanet_parser',
                  'use_bfloat16': False},
 'enable_summary': False,
 'eval': {'batch_size': 8,
          'eval_file_pattern': 'gs://apiss/coco/val-*',
          'eval_samples': 1000,
          'eval_timeout': None,
          'input_sharding': True,
          'min_eval_interval': 180,
          'type': 'box',
          'val_json_file': 'gs://apiss/coco/instances_val2017.json'},
 'fpn': {'batch_norm': {'batch_norm_epsilon': 0.0001,
                        'batch_norm_momentum': 0.997,
                        'batch_norm_trainable': True},
         'fpn_feat_dims': 256,
         'max_level': 7,
         'min_level': 3,
         'use_separable_conv': False},
 'model_dir': 'gs://apiss/coco/Model/cocopy_func11',
 'nasfpn': {'batch_norm': {'batch_norm_epsilon': 0.0001,
                           'batch_norm_momentum': 0.997,
                           'batch_norm_trainable': True},
            'dropblock': {'dropblock_keep_prob': None, 'dropblock_size': None},
            'fpn_feat_dims': 256,
            'max_level': 7,
            'min_level': 3,
            'num_repeats': 5,
            'use_separable_conv': False},
 'postprocess': {'max_level': 7,
                 'max_total_size': 100,
                 'min_level': 3,
                 'nms_iou_threshold': 0.5,
                 'pre_nms_num_boxes': 5000,
                 'score_threshold': 0.05,
                 'use_batched_nms': False},
 'predict': {'predict_batch_size': 8},
 'resnet': {'batch_norm': {'batch_norm_epsilon': 0.0001,
                           'batch_norm_momentum': 0.997,
                           'batch_norm_trainable': True},
            'dropblock': {'dropblock_keep_prob': None, 'dropblock_size': None},
            'resnet_depth': 50},
 'retinanet_head': {'anchors_per_location': 9,
                    'batch_norm': {'batch_norm_epsilon': 0.0001,
                                   'batch_norm_momentum': 0.997,
                                   'batch_norm_trainable': True},
                    'max_level': 7,
                    'min_level': 3,
                    'num_classes': 91,
                    'retinanet_head_num_convs': 4,
                    'retinanet_head_num_filters': 256,
                    'use_separable_conv': False},
 'retinanet_loss': {'box_loss_weight': 50,
                    'focal_loss_alpha': 0.25,
                    'focal_loss_gamma': 1.5,
                    'huber_loss_delta': 0.1,
                    'num_classes': 91},
 'retinanet_parser': {'aug_rand_hflip': True,
                      'aug_scale_max': 1.0,
                      'aug_scale_min': 1.0,
                      'autoaugment_policy_name': 'v0',
                      'match_threshold': 0.5,
                      'max_num_instances': 100,
                      'num_channels': 3,
                      'output_size': [640, 640],
                      'skip_crowd_during_training': True,
                      'unmatched_threshold': 0.5,
                      'use_autoaugment': False,
                      'use_bfloat16': False},
 'strategy_config': {'all_reduce_alg': None,
                     'num_gpus': 0,
                     'num_packs': 1,
                     'task_index': -1,
                     'tpu': 'node-1',
                     'worker_hosts': None},
 'strategy_type': 'tpu',
 'train': {'batch_size': 64,
           'checkpoint': {'path': '', 'prefix': ''},
           'frozen_variable_prefix': '(resnet\\d+/)conv2d(|_([1-9]|10))\\/',
           'input_sharding': False,
           'iterations_per_loop': 500,
           'l2_weight_decay': 0.0001,
           'learning_rate': {'init_learning_rate': 0.08,
                             'learning_rate_levels': [0.008, 0.0008],
                             'learning_rate_steps': [15000, 20000],
                             'type': 'step',
                             'warmup_learning_rate': 0.0067,
                             'warmup_steps': 500},
           'optimizer': {'momentum': 0.9, 'nesterov': True, 'type': 'momentum'},
           'total_steps': 22500,
           'train_file_pattern': '',
           'transpose_input': False},
 'type': 'retinanet',
 'use_tpu': True}
I0817 01:48:55.207570 140143557838656 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:48:55.254988 140143557838656 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:48:55.255242 140143557838656 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:48:55.255457 140143557838656 client.py:777] Refreshing access_token
I0817 01:48:55.348412 140143557838656 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:48:55.384383 140143557838656 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:48:55.384562 140143557838656 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:48:55.384666 140143557838656 client.py:777] Refreshing access_token
I0817 01:48:55.528092 140143557838656 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:48:55.566549 140143557838656 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:48:55.566850 140143557838656 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:48:55.566993 140143557838656 client.py:777] Refreshing access_token
I0817 01:48:55.712026 140143557838656 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:48:55.754162 140143557838656 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:48:55.754454 140143557838656 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:48:55.754593 140143557838656 client.py:777] Refreshing access_token
I0817 01:48:55.900172 140143557838656 remote.py:177] Entering into master device scope: /job:worker/replica:0/task:0/device:CPU:0
2021-08-17 01:48:55.900590: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2021-08-17 01:48:55.905169: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2249995000 Hz
2021-08-17 01:48:55.905442: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5d48700 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2021-08-17 01:48:55.905468: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2021-08-17 01:48:55.911151: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job worker -> {0 -> 10.116.237.58:8470}
2021-08-17 01:48:55.911323: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job localhost -> {0 -> localhost:42912}
2021-08-17 01:48:55.927334: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job worker -> {0 -> 10.116.237.58:8470}
2021-08-17 01:48:55.927386: I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:300] Initialize GrpcChannelCache for job localhost -> {0 -> localhost:42912}
2021-08-17 01:48:55.927866: I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:390] Started server with target: grpc://localhost:42912
INFO:tensorflow:Initializing the TPU system: node-1
I0817 01:48:55.928572 140143557838656 tpu_strategy_util.py:72] Initializing the TPU system: node-1
INFO:tensorflow:Clearing out eager caches
I0817 01:48:56.069632 140143557838656 tpu_strategy_util.py:100] Clearing out eager caches
INFO:tensorflow:Finished initializing TPU system.
I0817 01:49:00.983436 140143557838656 tpu_strategy_util.py:123] Finished initializing TPU system.
I0817 01:49:00.984860 140143557838656 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:49:01.025470 140143557838656 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:49:01.025761 140143557838656 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:49:01.025903 140143557838656 client.py:777] Refreshing access_token
I0817 01:49:01.111823 140143557838656 discovery.py:280] URL being requested: GET https://www.googleapis.com/discovery/v1/apis/tpu/v1/rest
I0817 01:49:01.154428 140143557838656 discovery.py:911] URL being requested: GET https://tpu.googleapis.com/v1/projects/concordia-sense-lab-project/locations/us-central1-a/nodes/node-1?alt=json
I0817 01:49:01.154601 140143557838656 transport.py:157] Attempting refresh to obtain initial access_token
I0817 01:49:01.154707 140143557838656 client.py:777] Refreshing access_token
INFO:tensorflow:Found TPU system:
I0817 01:49:01.239695 140143557838656 tpu_system_metadata.py:140] Found TPU system:
INFO:tensorflow:*** Num TPU Cores: 8
I0817 01:49:01.240034 140143557838656 tpu_system_metadata.py:141] *** Num TPU Cores: 8
INFO:tensorflow:*** Num TPU Workers: 1
I0817 01:49:01.240174 140143557838656 tpu_system_metadata.py:142] *** Num TPU Workers: 1
INFO:tensorflow:*** Num TPU Cores Per Worker: 8
I0817 01:49:01.240300 140143557838656 tpu_system_metadata.py:144] *** Num TPU Cores Per Worker: 8
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:CPU:0, CPU, 0, 0)
I0817 01:49:01.240436 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:CPU:0, CPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
I0817 01:49:01.240674 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:localhost/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:CPU:0, CPU, 0, 0)
I0817 01:49:01.240856 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:CPU:0, CPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:0, TPU, 0, 0)
I0817 01:49:01.241076 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:0, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:1, TPU, 0, 0)
I0817 01:49:01.241251 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:1, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:2, TPU, 0, 0)
I0817 01:49:01.241386 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:2, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:3, TPU, 0, 0)
I0817 01:49:01.241519 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:3, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:4, TPU, 0, 0)
I0817 01:49:01.241684 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:4, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:5, TPU, 0, 0)
I0817 01:49:01.241879 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:5, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:6, TPU, 0, 0)
I0817 01:49:01.242003 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:6, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:7, TPU, 0, 0)
I0817 01:49:01.242146 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU:7, TPU, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU_SYSTEM:0, TPU_SYSTEM, 0, 0)
I0817 01:49:01.242285 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:TPU_SYSTEM:0, TPU_SYSTEM, 0, 0)
INFO:tensorflow:*** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
I0817 01:49:01.242459 140143557838656 tpu_system_metadata.py:146] *** Available Device: _DeviceAttributes(/job:worker/replica:0/task:0/device:XLA_CPU:0, XLA_CPU, 0, 0)
I0817 01:49:01.243001 140143557838656 main.py:135] Eval num_replicas_in_sync 8 num_workers 1 is_multi_host False
I0817 01:49:31.076369 140143557838656 distributed_executor.py:607] Starting to evaluate.
I0817 01:49:32.203436 140143557838656 distributed_executor.py:615] Checkpoint file gs://apiss/coco/Model/cocopy_func11/ctl_step_5000.ckpt-10 found and restoring from checkpoint
WARNING:tensorflow:AutoGraph could not transform <function InputFn.__call__.<locals>.<lambda> at 0x7f75400c3f28> and will run it as-is.
Cause: Unable to identify source code of lambda function <function InputFn.__call__.<locals>.<lambda> at 0x7f75400c3f28>. It was defined on this line: from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
map_func, which must contain a single lambda with matching signature. To avoid ambiguity, define each lambda in a separate expression.
W0817 01:49:36.917421 140143557838656 ag_logging.py:146] AutoGraph could not transform <function InputFn.__call__.<locals>.<lambda> at 0x7f75400c3f28> and will run it as-is.
Cause: Unable to identify source code of lambda function <function InputFn.__call__.<locals>.<lambda> at 0x7f75400c3f28>. It was defined on this line: from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
map_func, which must contain a single lambda with matching signature. To avoid ambiguity, define each lambda in a separate expression.
I0817 01:49:43.783425 140143557838656 detection_executor.py:123] Running evaluation after step: 5000.
I0817 01:52:30.253574 140143557838656 coco_evaluator.py:151] Using annotation file: gs://apiss/coco/instances_val2017.json
I0817 01:52:54.684378 140143557838656 detection_executor.py:137] Step: [5000] Validation metric = {'AP': 0.24586119, 'AP50': 0.40423423, 'AP75': 0.26471335, 'APs': 0.10256057, 'APm': 0.31888944, 'APl': 0.3903242, 'ARmax1': 0.23876402, 'ARmax10': 0.38293582, 'ARmax100': 0.40453994, 'ARs': 0.16362847, 'ARm': 0.49101698, 'ARl': 0.58822423}
2021-08-17 01:52:54.684800: W tensorflow/core/distributed_runtime/eager/remote_tensor_handle_data.cc:75] Unable to destroy remote tensor handles. If you are running a tf.function, it usually indicates some op in the graph gets an error: 2 root error(s) found.
  (0) Cancelled: {{function_node __inference_test_step_89460}} Function was cancelled before it was started
  (1) Out of range: {{function_node __inference_test_step_89460}} End of sequence
	 [[{{node IteratorGetNext_6}}]]
0 successful operations.
7 derived errors ignored.
I0817 01:52:54.685314 140143557838656 distributed_executor.py:622] Step: 5000 evalation metric = {'AP': 0.24586119, 'AP50': 0.40423423, 'AP75': 0.26471335, 'APs': 0.10256057, 'APm': 0.31888944, 'APl': 0.3903242, 'ARmax1': 0.23876402, 'ARmax10': 0.38293582, 'ARmax100': 0.40453994, 'ARs': 0.16362847, 'ARm': 0.49101698, 'ARl': 0.58822423}.
I0817 01:52:55.516526 140143557838656 main.py:168] Final eval metric AP: 0.245861
I0817 01:52:55.516835 140143557838656 main.py:168] Final eval metric AP50: 0.404234
I0817 01:52:55.516914 140143557838656 main.py:168] Final eval metric AP75: 0.264713
I0817 01:52:55.516991 140143557838656 main.py:168] Final eval metric APs: 0.102561
I0817 01:52:55.517074 140143557838656 main.py:168] Final eval metric APm: 0.318889
I0817 01:52:55.517146 140143557838656 main.py:168] Final eval metric APl: 0.390324
I0817 01:52:55.517202 140143557838656 main.py:168] Final eval metric ARmax1: 0.238764
I0817 01:52:55.517264 140143557838656 main.py:168] Final eval metric ARmax10: 0.382936
I0817 01:52:55.517327 140143557838656 main.py:168] Final eval metric ARmax100: 0.404540
I0817 01:52:55.517409 140143557838656 main.py:168] Final eval metric ARs: 0.163628
I0817 01:52:55.517485 140143557838656 main.py:168] Final eval metric ARm: 0.491017
I0817 01:52:55.517540 140143557838656 main.py:168] Final eval metric ARl: 0.588224
loading annotations into memory...
Done (t=0.49s)
creating index...
index created!
creating index...
index created!
Running per image evaluation...
Evaluate annotation type *bbox*
DONE (t=18.02s).
Accumulating evaluation results...
DONE (t=1.88s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.246
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets=100 ] = 0.404
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets=100 ] = 0.265
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.103
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.319
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.390
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=  1 ] = 0.239
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 10 ] = 0.383
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets=100 ] = 0.405
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= small | maxDets=100 ] = 0.164
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets=100 ] = 0.491
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets=100 ] = 0.588
2021-08-17 01:52:56.703668: W ./tensorflow/core/distributed_runtime/eager/destroy_tensor_handle_node.h:79] Ignoring an error encountered when deleting remote tensors handles: Invalid argument: Unable to find a context_id matching the specified one (6845543869327027883). Perhaps the worker was restarted, or the context was GC'd?
Additional GRPC error information:
{"created":"@1629165176.703481331","description":"Error received from peer","file":"external/grpc/src/core/lib/surface/call.cc","file_line":1039,"grpc_message":"Unable to find a context_id matching the specified one (6845543869327027883). Perhaps the worker was restarted, or the context was GC'd?","grpc_status":3}
